config/prow_setup.md

```diff
@@ -69,7 +69,7 @@
 1. Make sure that _Knative Robots_ is an Admin of the repo.
 
 1. Add the new repo to [config_knative.yaml](./prow/config_knative.yaml),
-   without any presubmits. Run `make config` to regenerate
+   without any presubmits. Check the top-level section `presubmits:` and `periodics:` for blueprints for what to add. Then run `make config` to regenerate
```

Comment:
@chizhg I'm not sure what is meant "without any presubmits". Actually you _have to_ add a new section to `presubmits:`, or ? I was a bit confused what to add initially, so I just tried to copy over the config for `knative/client` (but overlooked the periodics part). A bit clarification (maybe even with an example ?) would be cool here.

Also, a link to reference documentation for the possible fields would be awesome (if such documentation exists).

Reply:
You can ignore that "without any presubmits" part. The doc is assuming you will create two PRs, one is setting up Prow for a new repo, and the next one is adding Prow job for the repo, but one PR that contains both is also fine.

Currently we don't have other reference doc for this, but thanks for helping us identify this gap, we will definitely make some improvements for the docs.